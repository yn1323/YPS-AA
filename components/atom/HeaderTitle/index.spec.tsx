import { render } from 'enzyme'
import React from 'react'
import { act } from 'react-test-renderer'
import Component from '.'

test(Component.name, async () => {
  let component
  await act(async () => {
    component = await render(<Component />)
  })

  expect(component).toMatchSnapshot()
})
