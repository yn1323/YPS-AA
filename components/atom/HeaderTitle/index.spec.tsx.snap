// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle 1`] = `
<h6
  className="MuiTypography-root HeaderTitle__TitleText-sc-ufp3wy-0 eAfaAT MuiTypography-h6"
/>
`;
